Games/core/Player.js

```diff
@@ -258,6 +258,36 @@ module.exports = class Player {
         return will;
     }
 
+    // Checks that player has voted in all meetings except the vegkick meeting.
+    // This function is used during the vegkick meeting, so vegkickmeeting should not be undefined.
+    hasVotedInAllMeetings() {
+        let allMeetings = this.getMeetings();
+        let vegKickMeetingId = this.getVegKickMeeting()?.id;
+
+        for (let meeting of allMeetings) {
+            if (meeting.id === vegKickMeetingId) {
+                continue;
+            }
+
+            if (meeting.finished) {
+                continue;
+            }
+
+            // player has not voted
+            if (meeting.members[this.id].canVote &&
+                meeting.members[this.id].canUpdateVote && 

```

Comment:
to be consistent,         
// Veg players who didn't vote
should also check if canUpdateVote is true